scripts/import_from_biosamples.py

```diff
@@ -262,6 +262,29 @@ def fetch_records_by_project_via_etag(etags):
     logger.info(f"Finish comparing etags and retrieving necessary records at {datetime.datetime.now()}")
 
 
+def unify_field_names(biosample):
+    """
+    BioSamples started to unify the fields name to make sure 1) all lowercase and 2) no underscores
+    all lowercase conflicts with FAANG ruleset, e.g. material in BioSamples and Material in FAANG ruleset
+    this method provides an automatical way to convert to FAANG standard
+    :param biosample: biosample record retrieved from BioSamples API
+    :return: converted BioSamples record
+    """
+    mapped_fields = {
+        "material": "Material",
+        "sex": "Sex",
+        "organism": "Organism"
+    }
+    for field_name in mapped_fields.keys():
```

Comment:
This could be simplified to:
for field_name, new_field_name in mapped_fields.items():
    if field_name in biosample['characteristics'] and new_field_name not in biosample['characteristics']:
        etc...